.changes/next-release/feature-6e6f3f94-ccf7-4c5d-89e5-e56c4fdddbc1.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "User enters/types \"delete me\" instead of the resource name to confirm deletion of resource."
```

Comment:
I'm not the best at writing changelog entries, but in general it should be written in a somewhat consistent style matching the other entries. So instead of `User enters` it should be something like `Deleting resources requires typing "delete me" instead of the resource name`